portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,284 @@
+        </div>
+    </div>
+    <div class="col-md-6">
+        <div class="card">
+            <img class="card-img-top" src="images/wildfire.jpg">
+            <div class="card-body">
+                <h4 class="card-title">UCB Research</h4>
+                <p class="card-text">A project invovling using small devices
+                     to detect fire as early as possible and request 
+                    satellite images.</p>
+                <a href="projects.html" class="btn btn-outline-secondary">More details</a>
+            </div>
+        </div>
+    </div>
+</div>
+</div>
+
+
+<!--- Hobbies header -->
+<a id="hobbies"></a>
+<div class="container-fluid padding">
+<div class="row welcome text-center">
+    <div class="col-12">
+        <h1 class="display-4">Hobbies</h1>
+    </div>
+    <hr>
+</div>
+</div>
+
+<!--- Golfing hobby -->
+<div class="container-fluid padding">
+<div class="row padding">
+    <div class="col-lg-6">
+        <h2>Golfing</h2>
+        <p> One thing that I enjoy to do the most is golfing in my free time. Growing
+          up, my father was an avid golfer, and he would bring me to the range to 
+          practice once in a while. However, joining the golf team in high school 
+          was when I really started to love golf. </p>
+        <p> During times when I am stressed
+          or tense, I'd be able to walk onto the golf course to relax and find some
+          serenity. Even in college, I would go to the course once in a while with 
+          friends to mess around. And even in quarantine, I would go to my friend's
+          house to play golf on his golf simulator. </p>
+    </div>
+    <div class="col-lg-6">
+        <img src="images/golf.jpg" class="img-fluid">
+    </div>
+</div>
+</div>
+
+<!--- Academics Background -->
+<div class="container-fluid padding">
+<div class="row padding">
+    <div class="col-lg-6">
+        <img src="images/youtube.png" class="img-fluid">
+    </div>
+    <div class="col-lg-6">
+        <h2>Youtube</h2>
+        <p> Another thing that I spend a lot of time doing is watching YouTube. It is
+          scary to me how good the recommendation algorithm is on YouTube because
+          I cannot seem to resist the content presented to me. I don't have a very
+          particular type of videos that I watch, but if you look through my watch
+          history, there will be fishing videos, political commentary videos, and 
+          even some episodes of Family Guy. </p>
+        <p> It was quite funny to me that I was 
+          originally supposed to work on a load test for YouTube over the summer. 
+          My rationale was: if I already spend this much time on it, I might as well
+          contribute to it myself. </p>
+    </div>
+</div>
+</div>
+
+<button onclick="getFetch()">Fetch!</button>
+<div class="col-lg-6" id="fetch-container"></div>
+
+<form action="/data" method="POST">
+
+      <p>Enter your name and your comment separated by a comma:</p>
```

Comment:
what happens if there are other commas in the comment?

Reply:
restructured the comments input structure